metron-interface/metron-config/src/app/sensors/sensor-threat-triage/rule-editor/sensor-rule-editor.component.spec.ts

```diff
@@ -84,17 +84,20 @@ describe('Component: SensorRuleEditorComponent', () => {
         component.riskLevelRule =  {name: 'rule1', rule: 'initial rule', score: '1', comment: ''};
         component.ngOnInit();
         component.onSave();
+        fixture.detectChanges();
```

Comment:
I'm not sure how this passed before, but this set of tests would fail after updating to Angular 7 because the `riskLevelRule` property's value was updated without calling change detection. Because we are directly and synchronously updating the component property's value in the test, we have to explicitly call change detection so the Angular testing environment knows of the value update.